Tests/MapboxNavigationTests/InstructionsBannerViewSnapshotTests.swift

```diff
@@ -441,21 +457,20 @@ class InstructionsBannerViewSnapshotTests: InstructionBannerTest {
         ]
         
         let secondaryInstruction: [VisualInstruction.Component] = [
-            .image(image: .init(imageBaseURL: nil),
-                   alternativeText: .init(text: "SLE",
-                                          abbreviation: nil,
-                                          abbreviationPriority: nil)),
+            .exitCode(text: .init(text: "15",
+                                  abbreviation: nil,
+                                  abbreviationPriority: nil)),
             .delimiter(text: .init(text: "/",
                                    abbreviation: nil,
                                    abbreviationPriority: nil)),
             .image(image: .init(imageBaseURL: nil),
-                   alternativeText: .init(text: "TPE",
+                   alternativeText: .init(text: "CTE",
                                           abbreviation: nil,
                                           abbreviationPriority: nil)),
         ]
         
         instructionsBannerView.primaryLabel.showHighlightedTextColor = true
-        instructionsBannerView.secondaryLabel.showHighlightedTextColor = true
+        instructionsBannerView.secondaryLabel.showHighlightedTextColor = false
 
         CustomDayStyle().apply()
         window.addSubview(instructionsBannerView)
```

Comment:
We could also remove the `window` and the `window.addSubview(_:)`from both `testGenericShieldAndExitViewWithCustomDayStyle` and `testGenericShieldAndExitViewHighlightColor`, as well as other unit tests in `InstructionsBannerViewSnapshotTests` . Because it only provides one phone window and we should test it on different phone and iOS version.